templates/shortcode-receipt.php

```diff
@@ -77,195 +71,263 @@
 		</tr>
 		<?php endif; ?>
 
-		<?php if ( ! empty( $order->get_fees() ) ) : ?>
-		<tr>
-			<td><strong><?php esc_html_e( 'Fees', 'easy-digital-downloads' ); ?>:</strong></td>
-			<td>
-				<ul class="edd_receipt_fees">
-				<?php foreach ( $order->get_fees() as $fee ) : ?>
-					<li>
-						<?php if ( 'order_item' === $fee->object_type || ! empty( $fee->description ) ) : ?>
-							<span class="edd_fee_label">
-								<?php
-								if ( 'order_item' === $fee->object_type ) {
-									$order_item = edd_get_order_item( $fee->object_id );
-									if ( $order_item ) {
-										echo esc_html( $order_item->product_name ) . ':';
-									}
-								}
-								?>
-							<?php if ( ! empty( $fee->description ) ) : ?>
-								<?php echo esc_html( $fee->description ); ?>
-							<?php endif; ?>
-							</span>
-							<span class="edd_fee_sep">&nbsp;&ndash;&nbsp;</span>
-						<?php endif; ?>
-						<span class="edd_fee_amount"><?php echo esc_html( edd_currency_filter( edd_format_amount( $fee->subtotal ) ) ); ?></span>
-					</li>
-				<?php endforeach; ?>
-				</ul>
-			</td>
-		</tr>
+		<?php if ( filter_var( $edd_receipt_args['price'], FILTER_VALIDATE_BOOLEAN ) && $order->subtotal > 0 ) : ?>
+			<tr>
+				<td><strong><?php esc_html_e( 'Subtotal', 'easy-digital-downloads' ); ?>:</strong></td>
+				<td>
+					<?php echo esc_html( edd_payment_subtotal( $order->id ) ); ?>
+				</td>
+			</tr>
+		<?php endif; ?>
+
+		<?php
+		if ( filter_var( $edd_receipt_args['discount'], FILTER_VALIDATE_BOOLEAN ) ) :
+			$discounts = $order->get_discounts();
+			if ( $discounts ) :
+				$label = _n( 'Discount', 'Discounts', count( $discounts ), 'easy-digital-downloads' );
+				?>
+				<tr>
+					<td colspan="2"><strong><?php echo esc_html( $label ); ?>:</strong></td>
+				</tr>
+				<?php
+				foreach ( $discounts as $discount ) {
```

Comment:
Can you rename `$discount` to either `$order_discount` or `$order_adjustment`? I just think that will make it clear that `$discount` is NOT an `EDD_Discount` object.